lib/Dialect/LWE/Conversions/LWEToOpenfhe/LWEToOpenfhe.cpp

```diff
@@ -400,16 +400,13 @@ struct LWEToOpenfhe : public impl::LWEToOpenfheBase<LWEToOpenfhe> {
         // TODO (#1193): Extend "common lwe" to support ctxt-ptxt ops
 
         // AddPlain
-        ConvertCiphertextPlaintextOp<bgv::AddPlainOp, openfhe::AddPlainOp>,
-        ConvertCiphertextPlaintextOp<ckks::AddPlainOp, openfhe::AddPlainOp>,
+        ConvertCiphertextPlaintextOp<lwe::RAddOp_Plain, openfhe::AddPlainOp>,
 
         // SubPlain
-        ConvertCiphertextPlaintextOp<bgv::SubPlainOp, openfhe::SubPlainOp>,
-        ConvertCiphertextPlaintextOp<ckks::SubPlainOp, openfhe::SubPlainOp>,
+        ConvertCiphertextPlaintextOp<lwe::RSubOp_Plain, openfhe::SubPlainOp>,
```

Comment:
You have defined sub_plain in LWE to allow the plaintext in either operand, but IIRC OpenFHE requires it is only in one operand. Could you double-check this, and then maybe this needs a custom rule or canonicalization pattern to ensure the plaintext operand is in the right place?

Reply:
BGV_SubPlainOp also allows pt in either operand, openfhe has two overloads for `cc->EvalSub` that accepts both of them. It is Lattigo that only accepts `evaluator.Sub(ct, pt)`, but that reordering is handled in `LWEToLattigo` by `Add(Negate(ct), pt)`, where Negate is only present in `lattigo` dialect, so it should not be a concern for `lwe` dialect.